src/pages/MainHome.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useRef } from 'react';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+interface Article {
+  articles: [
+    {
+      slug: string;
+      title: string;
+      description: string;
+      body: string;
+      tagList: [];
+      createdAt: string;
+      updatedAt: string;
+      favorited: boolean;
+      favoritesCount: number;
+      author: {
+        username: string;
+        bio: string;
+        image: string;
+        following: boolean;
+      };
+    },
+  ];
+}
+
+interface Articles {
+  articles: [];
+}
+
+const MainHome = () => {
+  // const [articles, setArticles] = useState<Articles>({
+  //   slug: '',
+  //   title: '',
+  //   description: '',
+  //   body: '',
+  //   tagList: [],
+  //   createdAt: '',
+  //   updatedAt: '',
+  //   favorited: false,
+  //   favoritesCount: 0,
+  //   author: {
+  //     username: '',
+  //     bio: '',
+  //     image: '',
+  //     following: false,
+  //   },
+  // });
+  const [articles, setArticles] = useState<Article>();
+
+  const arrayArticle = useRef<Article>();
+  useEffect(() => {
+    axios.get('https://api.realworld.io/api/articles').then((res) => {
+      console.log('res', res.data.articles);
+      setArticles(res.data);
+    });
+  }, []);
+  if (articles !== undefined) {
+    if (articles.articles !== undefined)
+      console.log('articles', articles.articles.length);
+  }
```

Comment:
🍌 불필요한 console은 지우는 게 가독성에 도움이 될 것 같아요!

🥝 `articles.articles`가 존재할 때만 콘솔을 띄우고 싶은 거라면 맞는 코드지만, 그냥 띄우기만 하면 된다면 이렇게 해도 좋을 것 같습니다!
```suggestion
  console.log('articles', articles?.articles?.length);
```